### PR TITLE
[SYCL][UR][CUDA] Fix some issues in cuda adapter's urDeviceGetInfo.

### PR DESCRIPTION
**Pull request by aarongreig**

### CODE REVIEW

File: sycl/plugins/unified_runtime/ur/adapters/cuda/device.cpp [hunk 1]
```diff
@@ -315,7 +316,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
           "runtime.");
     }
 
-    return ReturnValue(uint32_t{Enabled});
```
[the remark this answers is not in the document]
Comment (callumfare) [reply sycl/plugins/unified_runtime/ur/adapters/cuda/device.cpp:318]: This should be ok now, adapters should return a `bool` and pi2ur now handles the conversion to `pi_bool` which is `uint32_t`

[the remark this answers is not in the document]
Comment (aarongreig) [reply sycl/plugins/unified_runtime/ur/adapters/cuda/device.cpp:318]: nice

**Review by jchlanda — APPROVED** [review]
I'd change the commit message to something more descriptive, maybe Use correct types in CUDA's urDeviceGetInfo?

**Comment by aarongreig** [comment]
Update the commit message, is there anyone else I should ping for more reviews/to get this merged @jchlanda?

**Comment by aarongreig** [comment]
weird CI failure:
```
fatal: detected dubious ownership in repository at '/__w/repo_cache/intel/llvm'
```

**Comment by dm-vodopyanov** [comment]
> weird CI failure:
> 
> ```
> fatal: detected dubious ownership in repository at '/__w/repo_cache/intel/llvm'
> ```

Just in case we have an issue on that: https://github.com/intel/llvm/issues/10324

**Pull request merged**